.ci/magician/github/reviewer_assignment.go

```diff
@@ -66,3 +67,24 @@ func FormatReviewerComment(newPrimaryReviewer string) string {
 	})
 	return sb.String()
 }
+
+var reviewerCommentRegex = regexp.MustCompile("@(?P<reviewer>[^,]*), a repository maintainer, has been assigned to review your changes.")
```

Comment:
We should definitely have a test that verifies that this matches the output of the function that formats this comment.

Reply:
Done